src/mudlet.cpp

```diff
@@ -122,7 +122,11 @@ const int mudlet::scmQDataStreamFormat_5_12 = 18;
 QPointer<TConsole> mudlet::mpDebugConsole = nullptr;
 QPointer<QMainWindow> mudlet::mpDebugArea = nullptr;
 bool mudlet::debugMode = false;
-const bool mudlet::scmIsDevelopmentVersion = !QByteArray(APP_BUILD).isEmpty();
+
+const bool mudlet::scmIsReleaseVersion = QByteArray(APP_BUILD).isEmpty();
+const bool mudlet::scmIsPublicTestVersion = QByteArray(APP_BUILD) == QStringLiteral("-public-test-build");
```

Comment:
Are you sure you need `QStringLiteral`...`)` around that raw string - it is being compared to a `QByteArray` which is **NOT** a `QString`...? :confused:

Reply:
There is an overload that makes it work: https://doc.qt.io/qt-5/qbytearray.html#operator-eq-eq